tests/run_nft_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+NETNS=nft-$(cat /proc/sys/kernel/random/uuid)
+
+function nsexec {
+  ip netns exec $NETNS $@
+}
+
+function cleanup {
+  ip netns delete "$NETNS"
+  exit 0
+}
+trap cleanup EXIT
+
+# create net namespace
+(ip netns ls | grep -Fx "$NETNS" 2>/dev/null) || ip netns add "$NETNS"
+
+nft --version;
+nsexec cargo test --verbose -- --ignored
```

Comment:
FYI, it should be much simpler to just use `unshare -n cargo ...` to create a new netns here, no need to manage any ip netns "names". You can also do `unshare -rn` as normal user (assuming unprivileged user namespaces are enabled, which is the default for most distros) so not even a need for sudo.
Because the netns is only tied to the process once the process exits the netns will be gone so you also can skip the clean-up logic as well